### ISSUE TITLE
Protonmail Bridge Compatibility

### ISSUE THREAD

**Issue by davidpgil** [deltachat/deltachat-desktop#669]
I just tried to setup Deltachat with my Protonmail address and I couldn't seem to get it to work. Does or will Deltachat support Protonmail? My understandign is that Deltachat would need to be compatible with [Protonmail Bridge](https://protonmail.com/bridge/) in particular.

I'm using Mac OS Mojave. I have been trying to adapt these instructions from the Protonmail site: [https://protonmail.com/bridge/thunderbird-mac#1](https://protonmail.com/bridge/thunderbird-mac#1)

**Comment by Simon-Laux** [comment]
`I just tried to setup Deltachat with my Protonmail address and I couldn't seem to get it to work`
Which exact steps did you take?
Did you use the advanced settings?

**Comment by Jikstra** [comment]
I don't know of anyone who used it in combination yet, but if it starts something like a small smtp/imap server wrapper/proxy, this could work. I don't use protonmail so i can't test, but in general support for protonmail would be cool.

**Comment by Simon-Laux** [comment]
Any updates on this?

**Comment by Jikstra** [comment]
We need to be able to allow self signed certificates which is currently not possible. @adbeniez is facing this problem too. https://github.com/deltachat/deltachat-desktop/issues/909

But mainly a core issue. Would like to close it here in desktop.

**Comment by hpk42** [comment]
self-signed certs relate to the TLS hostname validation issue --
https://github.com/deltachat/deltachat-core-rust/issues/203
both are about invalid certs.

**Comment by Jikstra** [comment]
Closed as this first needs to get added in core: deltachat/deltachat-core-rust#203

**Comment by kamilko** [comment]
This issue is still not resolved in 0.840.0-PREVIEW version.
On mac os Mojave with ProtonBridge latest, while DeltaChat is trying to connect to server (127.0.0.1) crashes.
Certificate checks is set to: Accept invalid certificates. (but any of the options not working)
After crash in log there is only warning as below. Crash occurs when progress bar is near end.

```
DeltaChat.dc.on (/Applications/DeltaChat.app/Contents/Resources/app.asar/src/main/deltachat/controller.js:243:11),
DeltaChat.emit (events.js:182:13),
handleEvent (/Applications/DeltaChat.app/Contents/Resources/app.asar/node_modules/deltachat-node/lib/deltachat.js:747:12),
DeltaChat.binding.dcn_set_event_handler (/Applications/DeltaChat.app/Contents/Resources/app.asar/node_modules/deltachat-node/lib/deltachat.js:35:7)
```

**Comment by adbenitez** [comment]
Hello, I think the certificate issue is fixed on 0.900
